Expecto/Expecto.Impl.fs

```diff
@@ -289,7 +289,7 @@ module Impl =
           let spirit =
             if summary.successful then "Success!" else String.Empty
           let commonAncestor =
-            let rec loop ancestor (descendants : string list) =
+            let rec loop (ancestor: string) (descendants : string list) =
```

Comment:
this change was to address an inference error because `ancestor` has a member indexed in an unknown way due to new overloads in .net 5.